pw-pbkdf2.c

```diff
@@ -399,15 +481,95 @@ static int pbkdf2_check(
 						  dk_len, iteration,
 						  PBKDF2_SALT_SIZE, salt_value,
 						  dk_len, input_dk_value);
+
+#elif HAVE_MOZNSS
+	SECStatus rv;
+	rv = NSS_NoDB_Init(".");
```

Comment:
https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/SSL_functions/sslfnc.html#1234224
> NSS_NoDB_Init isnot idempotent, so call it only once.

so, move to init_module().

Reply:
Done